build.sbt

```diff
@@ -66,6 +66,7 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
   .in(file("core"))
   .settings(stdSettings("zio-macros-core"))
   .settings(macroSettings())
+  .settings(libraryDependencies += "dev.zio" %% "zio-test" % zioVersion)
```

Comment:
Why `zio-test` dependency? As far as I can tell only `zio` core is needed.

Reply:
Yeah, was a typo